app/src/main/java/org/fossasia/openevent/app/common/data/models/Event.java

```diff
@@ -47,54 +52,58 @@
 
     @Id(LongIdHandler.class)
     @PrimaryKey
-    public long id;
+    public Long id;
     public String paymentCountry;
     public String paypalEmail;
     public String thumbnailImageUrl;
     public String schedulePublishedOn;
     public String paymentCurrency;
     public String organizerDescription;
-    public boolean isMapShown;
     public String originalImageUrl;
     public String onsiteDetails;
     public String organizerName;
-    public boolean canPayByStripe;
     public String largeImageUrl;
     public String timezone;
-    public boolean canPayOnsite;
     public String deletedAt;
     public String ticketUrl;
-    public boolean canPayByPaypal;
     public String locationName;
-    public boolean isSponsorsEnabled;
-    public boolean hasOrganizerInfo;
-    public boolean isSessionsSpeakersEnabled;
     public String privacy;
     public String codeOfConduct;
     public String state;
-    public double latitude;
-    public String startsAt;
     public String searchableLocationName;
-    public boolean isTicketingEnabled;
-    public boolean canPayByCheque;
     public String description;
     public String pentabarfUrl;
     public String xcalUrl;
     public String logoUrl;
     public String externalEventUrl;
-    public boolean isTaxEnabled;
     public String iconImageUrl;
     public String icalUrl;
     public String name;
-    public boolean canPayByBank;
-    public String endsAt;
     public String createdAt;
-    public double longitude;
     public String bankDetails;
     public String chequeDetails;
     public String identifier;
-
     public boolean isComplete;
+    public double latitude;
+    public double longitude;
+    public boolean canPayByStripe;
+    public boolean canPayByCheque;
+    public boolean canPayByBank;
+    public boolean canPayByPaypal;
+    public boolean canPayOnsite;
+    public boolean isSponsorsEnabled;
+    public boolean hasOrganizerInfo;
+    public boolean isSessionsSpeakersEnabled;
+    public boolean isTicketingEnabled;
+    public boolean isTaxEnabled;
+    public boolean isMapShown;
+
+    @JsonSerialize(using = ObservableStringSerializer.class)
+    @JsonDeserialize(using = ObservableStringDeserializer.class)
+    public ObservableString startsAt = new ObservableString();
+    @JsonSerialize(using = ObservableStringSerializer.class)
+    @JsonDeserialize(using = ObservableStringDeserializer.class)
+    public ObservableString endsAt = new ObservableString();
```

Comment:
This is a deprecated method of binding and warnings are displayed. Data binding developers will remove this method soon, they have advised to not use it

Reply:
@iamareebjamal CreateEvent as well as AboutEvent uses this method of binding and gives a warning. Both use the same DateService methods etc can I open a separate issue for this warning and use the same mechanism for both of them because changing them in one place will give error in other. What do you say?

Reply:
Yup :+1: